<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="com.facebook.TestCases.LoginPageTest" duration-ms="5273" started-at="2018-09-13T03:46:30Z" finished-at="2018-09-13T03:46:35Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="5273" started-at="2018-09-13T03:46:30Z" finished-at="2018-09-13T03:46:35Z">
      <class name="com.facebook.TestCases.LoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.facebook.TestCases.LoginPageTest@18c7aca]" name="setUp" is-config="true" duration-ms="30265" started-at="2018-09-13T03:46:00Z" finished-at="2018-09-13T03:46:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="verifyLoginTest()[pri:0, instance:com.facebook.TestCases.LoginPageTest@18c7aca]" name="verifyLoginTest" duration-ms="5035" started-at="2018-09-13T03:46:30Z" finished-at="2018-09-13T03:46:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
        <test-method status="FAIL" signature="verifyTitleTest()[pri:0, instance:com.facebook.TestCases.LoginPageTest@18c7aca]" name="verifyTitleTest" duration-ms="76" started-at="2018-09-13T03:46:35Z" finished-at="2018-09-13T03:46:35Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Facebook – log in or sign up] but found [Log in to Facebook | Facebook]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Facebook – log in or sign up] but found [Log in to Facebook | Facebook]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.facebook.TestCases.LoginPageTest.verifyTitleTest(LoginPageTest.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTitleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.facebook.TestCases.LoginPageTest@18c7aca]" name="tearDown" is-config="true" duration-ms="1066" started-at="2018-09-13T03:46:36Z" finished-at="2018-09-13T03:46:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.facebook.TestCases.LoginPageTest -->
    </test> <!-- Command line test -->
  </suite> <!-- com.facebook.TestCases.LoginPageTest -->
</testng-results>
